cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -0,0 +1,275 @@
+# Default values for a single rook-ceph cluster
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Namespace of the main rook operator
+operatorNamespace: rook-ceph
+
+# Ability to override ceph.conf
+configOverride:
+#configOverride: |
+#  [global]
+#  mon_allow_pool_delete = true
+#
+#  osd_pool_default_size = 3
+#  osd_pool_default_min_size = 2
+
+# Installs a debugging toolbox deployment
+toolbox:
+  enabled: false
+  image: rook/ceph
```

Comment:
Can we merge the tag with the image, rather than specifying a separate `tag` setting? This is the pattern we've adhered to other places in rook. It tends to keep things simpler.